templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +26,12 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}" class="btn btn-primary">UPDATE</a>
```

Comment:
should be links not buttons here: "Also, add columns Update, Delete, and add links for the updating page and deletion page for each manufacturer."